git/github/github.go

```diff
@@ -52,6 +53,7 @@ type CommitStatusAdapter struct {
 	State       string
 	Description string
 	Context     string
+	TargetURL   string
```

Comment:
I may be missing something, but I can't see where this value is set to the correct console URL in the current code?
I would expect the [FormatPipelineURL](https://github.com/redhat-appstudio/integration-service/blob/c25c29fb82da14a96095e3aeba6f0d17ee062f6c/status/format.go#L214) to be called somewhere in order to generate the URL to be set for this.

Reply:
I've added an assignment to update TestPipelineRunName  [here](https://github.com/kasemAlem/integration-service/blob/32590e8a4646c8800a029fffb1125763b3a6f939/status/reporter_github.go#L181)